src/Microsoft.DotNet.Interactive.Documents.Tests/DocumentFormatTestsBase.cs

```diff
@@ -11,9 +11,9 @@ protected DocumentFormatTestsBase()
     {
         DefaultKernelInfos = new KernelInfoCollection
         {
-            new("csharp", new[] { "cs", "C#", "c#" }),
-            new("fsharp", new[] { "fs", "F#", "f#" }),
-            new("pwsh", new[] { "powershell" }),
+            new("csharp", "csharp", new[] { "cs", "C#", "c#" }),
+            new("fsharp", "fsharp", new[] { "fs", "F#", "f#" }),
+            new("pwsh", "powershell", new[] { "powershell" }),
```

Comment:
We should use the correct language names here so the tests illustrate normal usage.